### ISSUE TITLE
Firefox mobile - Viewport not mobile friendly

### ISSUE THREAD

**Issue by mattpalermo** [asciidoctor/asciidoctor-browser-extension#212]
The browser extension seems to work great on firefox mobile! Awesome! But it seems that the viewport is not mobile friendly. Ie. It appears as though the meta viewport tag is not present in the html output. I can't do too much debugging right now without access to my computer. I will when I can.

**Comment by mojavelinux** [comment]
I agree. It should have:

```
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

just like the default Asciidoctor HTML output.

**Comment by mattpalermo** [comment]
Solved by #213. It should be available with the next release.
